src/app/Components/DoctorProfile/update-doctor/update-doctor.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```

Comment:
space between imports isn't needed

Reply:
Angular created this file in this way, it's not me. I think it's a proper way, because in other files are the same story